apps/pm-user/src/components/molecules/profileSummary/profilesummary.module.scss

```diff
@@ -1,4 +1,4 @@
-.profileSummery {
+.profileSummary {
   :global {
     .ant-image-img {
       border-radius: 12px;
```

Comment:
Filename is not in camel case.

Reply:
![image](https://user-images.githubusercontent.com/48716076/185735483-e98f200e-3b6d-4409-91ed-4c8fb7cd4ff4.png)

Reply:
ok

Reply:
I did it and uploaded it but not reflected in remote repo